pyomo/contrib/sensitivity_toolbox/examples/feedbackController.py

```diff
@@ -17,7 +17,7 @@
 # 
 # min J(u) = (1/2)*H*x^2(T)+\int_0^T ((1/4)*u^2(t)dt)
 
-from pyomo.environ import * 
+from pyomo.environ import ConcreteModel, Param, Var, Objective, Constraint, Suffix, value
```

Comment:
@blnicho @jsiirola - This is an example of when Transformation + SolverFactory were NOT imported from Environ. There are a handful of these - I'll go through and find them.